Patches/[O21] Outer Rim Galaxies/Patch_OuterRim_Weapons.xml

```diff
@@ -0,0 +1,687 @@
+							<label>edge</label>
+							<capacities>
+								<li>Cut</li>
+							</capacities>
+							<power>30</power>
+							<cooldownTime>2.34</cooldownTime>
+							<chanceFactor>1.33</chanceFactor>
+							<armorPenetrationBlunt>0.956</armorPenetrationBlunt>
+							<armorPenetrationSharp>15</armorPenetrationSharp>
+							<linkedBodyPartsGroup>Edge</linkedBodyPartsGroup>
+						</li>
+					</tools>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="O21_OR_VibroCleaver"]/statBases</xpath>
+				<value>
+					<Bulk>5</Bulk>
+					<MeleeCounterParryBonus>0.35</MeleeCounterParryBonus>				
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="O21_OR_VibroCleaver"]</xpath>
+				<value>
+					<equippedStatOffsets>
+						<MeleeCritChance>0.2</MeleeCritChance>
+						<MeleeParryChance>0.35</MeleeParryChance>
+						<MeleeDodgeChance>0.1</MeleeDodgeChance>	
+					</equippedStatOffsets>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="O21_OR_VibroGlaive"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>handle</label>
+							<capacities>
+								<li>Poke</li>
+							</capacities>
+							<power>2</power>
+							<cooldownTime>1.44</cooldownTime>
+							<armorPenetrationBlunt>0.425</armorPenetrationBlunt>
+							<linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>point</label>
+							<capacities>
+								<li>Stab</li>
+							</capacities>
+							<power>37</power>
+							<cooldownTime>2.88</cooldownTime>
+							<armorPenetrationBlunt>0.425</armorPenetrationBlunt>
+							<armorPenetrationSharp>17</armorPenetrationSharp>
+							<linkedBodyPartsGroup>Point</linkedBodyPartsGroup>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<label>edge</label>
+							<capacities>
+								<li>Cut</li>
+							</capacities>
+							<power>32</power>
+							<cooldownTime>2.64</cooldownTime>
+							<chanceFactor>1.33</chanceFactor>
+							<armorPenetrationBlunt>0.956</armorPenetrationBlunt>
+							<armorPenetrationSharp>15</armorPenetrationSharp>
+							<linkedBodyPartsGroup>Edge</linkedBodyPartsGroup>
+						</li>
+					</tools>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="O21_OR_VibroGlaive"]/statBases</xpath>
+				<value>
+					<Bulk>3.5</Bulk>
+					<MeleeCounterParryBonus>0.35</MeleeCounterParryBonus>				
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="O21_OR_VibroGlaive"]</xpath>
+				<value>
+					<equippedStatOffsets>
+						<MeleeCritChance>0.4</MeleeCritChance>
+						<MeleeParryChance>0.7</MeleeParryChance>
+						<MeleeDodgeChance>0.2</MeleeDodgeChance>	
+					</equippedStatOffsets>
+				</value>
+			</li>
+			
+			<!-- Lightsaber patch from fully functional lightsaber mod -->
```

Comment:
One of these lightsaber operation is failing. I'm not sure specifically which.

Reply:
Should be fixed now